### PR TITLE
Provide README information for amtool

### PR DESCRIPTION
**Pull request by Kellel**
Amtool has been bundled with alertmanager for a while now. It's probably
far past the acceptable time to add information to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,9 +166,96 @@ receivers:
   - service_key: <team-DB-key>
 ```
 
+## Amtool
+
+`amtool` is a cli tool for interacting with the alertmanager api. It is bundled with all releases of alertmanager,
```
Comment (stuartnelson3) [inline README.md]: The ending `,` should probably be a `.`

File: README.md [hunk 2]
```diff
@@ -166,9 +166,96 @@ receivers:
...
+            Require a comment on silence creation
+
+    output
+            Set a default output type. Options are (simple, extended, json)
+```
+
 ## High Availability
 
-> Warning: High Availablility is under active development
+> Warning: High Availability is under active development
```
Comment (stuartnelson3) [inline README.md]: Hm, think I might prefer the old spelling.

File: README.md [hunk 3]
```diff
@@ -166,9 +166,96 @@ receivers:
...
+ID                                    Matchers              Ends At                  Created By  Comment
+b3ede22e-ca14-4aa0-932c-ca2f3445f926  alertname=Test_Alert  2017-08-02 19:54:50 UTC  kellel
+```
+
+Expire a silence
+```
+$ amtool silence expire b3ede22e-ca14-4aa0-932c-ca2f3445f926
+```
+
+### Config
```
Comment (stuartnelson3) [inline README.md]: I think it makes more sense to have this right after the install instructions. Thoughts?

File: README.md [hunk 4]
```diff
@@ -166,9 +166,96 @@ receivers:
...
+    author
+            Set a default author value for new silences. If this argument is not specified then the username will be used
+
+    comment_required
+            Require a comment on silence creation
+
+    output
+            Set a default output type. Options are (simple, extended, json)
+```
+
```
Comment (stuartnelson3) [inline README.md]: Can you add an annotated example config file instead?

File: README.md [hunk 5]
```diff
@@ -166,9 +166,96 @@ receivers:
...
+```
+$ amtool silence add alertname=Test_Alert
+b3ede22e-ca14-4aa0-932c-ca2f3445f926
+```
+
+View silences
+```
+$ amtool silence query
+ID                                    Matchers              Ends At                  Created By  Comment
+b3ede22e-ca14-4aa0-932c-ca2f3445f926  alertname=Test_Alert  2017-08-02 19:54:50 UTC  kellel
```
Comment (stuartnelson3) [inline README.md]: An example with filtering would be good.

File: README.md [hunk 6]
```diff
@@ -166,9 +166,96 @@ receivers:
...
+```
+
+Expire a silence
+```
+$ amtool silence expire b3ede22e-ca14-4aa0-932c-ca2f3445f926
```
Comment (stuartnelson3) [inline README.md]: might be helpful to show how to query with the `-q` option. I think that combined with the filtering is a powerful tool to remove a group of silences, and might be good to mention.

File: README.md (hunk 3 above)
Comment (Kellel) [inline README.md]: ya I guess that makes sense. I'm kinda of the opinion that you should show me the cool stuff as soon as possible before I get bogged down in how to set something up. But in this case I guess unless you are running alertmanager on the default port on your local machine you might get confused.

File: README.md (hunk 4 above)
Comment (Kellel) [inline README.md]: thats a great idea. I like being able to just copy and paste to start using things

**Comment by daachi** [comment]
It's probably because i don't know what the hell I'm doing, but `go get github.com/prometheus/alertmanager/cmd/amtool` fails for me. I ended up just building `alertmanager` to get `amtool`. _**After**_ I had `alertmanager` checked out in my `$GOPATH` I was able to `go get ...`.

Thanks for writing this doc.

File: README.md [hunk 7]
```diff
@@ -166,9 +166,120 @@ receivers:
...
 
+## Amtool
+
+`amtool` is a cli tool for interacting with the alertmanager api. It is bundled with all releases of alertmanager.
+
+### Install
+
+Alternatively you can install with:
+```
+go get github.com/prometheus/alertmanager/cmd/amtool
```
Comment (brancz) [inline README.md]: As we ship `amtool` with every release it's most appropriate to describe that it is included in the release tarball IMO, as we have absolutely no guarantee regarding compatibility across versions. So it's safest to just use the amtool that was shipped with the release you're using.

**Comment by stuartnelson3** [comment]
I removed the alertmanager repo from my `$GOPATH` and was able to successfully `go get github.com/prometheus/alertmanager/cmd/amtool`, so I'm not sure what you experienced.

**Pull request merged**